src/Servers/IIS/IIS/test/Common.FunctionalTests/FrebTests.cs

```diff
@@ -36,7 +49,6 @@ public static IList<FrebLogItem> FrebChecks()
         }
 
         [ConditionalFact]
-        [MaximumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10_20H2, SkipReason = "Shutdown hangs https://github.com/dotnet/aspnetcore/issues/25107")]
```

Comment:
?

Reply:
Yeah I was using this branch to test stuff, I thought i was done, so push had some junk, reverted now